http/vibe/http/proxy.d

```diff
@@ -226,6 +260,8 @@ final class HTTPReverseProxySettings {
 	/// ditto
 	@property void destinationPort(ushort port) { destination.port = port; }
 
+	/// Whether the proxy should act as forward or reverse proxy
+	bool reverseProxyMode = false;
```

Comment:
It would also be nice to convert this into a `enum ProxyMode { forward, reverse }` and add a constructor that takes a mode, so that forward/reverse becomes an explicit decision. A default constructor would be needed to avoid breaking existing code, but that can be deprecated/removed later down the road.